lib/ui/painting/image_decoder_impeller.h

```diff
@@ -33,7 +33,9 @@ class ImageDecoderImpeller final : public ImageDecoder {
 
   static std::shared_ptr<SkBitmap> DecompressTexture(
       ImageDescriptor* descriptor,
-      SkISize target_size);
+      SkISize target_size,
+      int32_t max_width,
```

Comment:
Why not just ISize instead of the two args?

Reply:
Done.